x/ccv/provider/handler_test.go

```diff
@@ -103,7 +103,7 @@ func TestAssignConsensusKeyForConsumerChain(t *testing.T) {
 			tc.setup(ctx, k, mocks)
 
 			msg, err := types.NewMsgAssignConsumerKey(tc.chainID,
-				testValProvider.SDKValAddress(), testValConsumer.SDKPubKey(),
+				testValProvider.SDKValOpAddress(), testValConsumer.ConsensusSDKPubKey(),
```

Comment:
Are we sure `NewMsgAssignConsumerKey` should accept a provider op address and a consumer cons pub key? Seems like it'd accept two cons or two operator addresses. Although I'm not that familiar myself 

CC @mpoke

Reply:
I think that `NewMsgAssignConsumerKey` mimics `NewMsgCreateValidator` in order to work with a similar CLI.

>  Seems like it'd accept two cons or two operator addresses

Actually only the former would work since consumer chains don't store operator addresses.

Reply:
Got it, I think my confusion is just stemming from sdk's address/key semantics. This API is good